phd-tests/framework/src/artifacts/mod.rs

```diff
@@ -13,12 +13,15 @@ pub use store::Store as ArtifactStore;
 
 pub const DEFAULT_PROPOLIS_ARTIFACT: &str = "__DEFAULT_PROPOLIS";
 
-#[derive(Clone, Debug, Serialize, Deserialize)]
+pub const CRUCIBLE_DOWNSTAIRS_ARTIFACT: &str = "crucible-downstairs";
```

Comment:
nit: although we're not currently supporting arbitrary Crucible artifacts, I'd consider using the two-underscores-and-screaming-snake-case convention as on line 14 for this as well. That said this is mostly stylistic preference--it's useful only to the extent that it very gently tips the scales against having a conflicting artifact in the artifact TOML, and if there *is* one it's not a hard conflict to detect

Reply:
can do!